Source/WebCore/platform/graphics/gstreamer/WebKitMediaSourceGStreamer.h

```diff
@@ -27,6 +27,7 @@
 #include "MediaSource.h"
 #include "SourceBufferPrivateClient.h"
 #include "SourceBufferPrivate.h"
+#include "MediaPlayerPrivateGStreamer.h"
```

Comment:
Maybe the MediaPlayerPrivateGStreamer class could be forward-declared here and the include moved to the cpp file.